### PR TITLE
demo/series-linecap

### PR DESCRIPTION
**Pull request by kamil-musialowski**
API: Added demo for linecap.

https://api.highcharts.com/highcharts/plotOptions.series.linecap

### CODE REVIEW

File: ts/Core/Series/SeriesDefaults.ts [hunk 1]
```diff
@@ -451,6 +451,9 @@ const seriesDefaults: SeriesOptions = {
     /**
      * The line cap used for line ends and line joins on the graph.
      *
+     * @sample {highcharts|highstock} highcharts/series/linecap/
```
Comment (raf18seb) [inline ts/Core/Series/SeriesDefaults.ts:454]: Is it necessary? Could you generate local docs and see if it's working the same way without it?
```{highcharts|highstock}```

File: samples/highcharts/series/linecap/demo.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+    series: [{
+        data: [2, 5, 2, 3, 6],
+        linecap: 'round' // default
+    }, {
+        data: [3, 6, 3, 4, 7],
+        linecap: 'square'
+    }]
+
+});
```
Comment (raf18seb) [inline samples/highcharts/series/linecap/demo.js:25]: I would personally add this demo to the `highcharts\samples\highcharts\plotoptions` folder

**Review by raf18seb — CHANGES_REQUESTED** [review]
Thanks for a clear demo!
One question and small comment, please see below

File: samples/highcharts/series/linecap/demo.js (hunk 2 above)
Comment (TorsteinHonsi) [reply samples/highcharts/series/linecap/demo.js:25]: The `plotoptions` folder is pretty cramped... So I would prefer adding a new folder for line series on the pattern `series-{{seriestype}}`:

`/samples/highcharts/series-line/linecap`

File: ts/Core/Series/SeriesDefaults.ts (hunk 1 above)
Comment (kamil-musialowski) [reply ts/Core/Series/SeriesDefaults.ts:454]: You're right! It's not necessary, everything is working fine without `{highcharts|highstock}`.

**Review by raf18seb — APPROVED** [review]
Thanks!

**Pull request merged**